common/os/macos.go

```diff
@@ -0,0 +1,15 @@
+package os
+
+type MacOS struct{}
+
+func NewMacOS() *MacOS {
+	return &MacOS{}
+}
+
+func (*MacOS) GetServiceManager() *serviceManager {
+	return &serviceManager{restartCmd: []string{"launchctl stop com.datadoghq.agent", "launchctl start com.datadoghq.agent"}}
+}
+
+func (*MacOS) GetConfigPath() string { return "~/.datadog-agent/datadog.yaml" }
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement. -->
**🥜 nitpick**
Wonder if this should be `GetAgentConfigPath`. Should we also have a `GetAgentConfdPath` for integration config paths ?

Reply:
I didn't change the name of `GetConfigPath` in this PR but indeed `GetAgentConfigPath` is more relevant.